src/control/XournalMain.cpp

```diff
@@ -19,6 +19,7 @@
 #include <boost/algorithm/string/predicate.hpp>
 #include <boost/filesystem.hpp>
 #include <boost/locale.hpp>
+#undef ENABLE_NLS
```

Comment:
Does translation still work with this change?
Or do you fully disable translation?